distribution/docs/src/main/resources/_contents/installing-contents.adoc

```diff
@@ -56,250 +54,216 @@ echo $JAVA_HOME
-----
-${branding-lowercase}${at-symbol}local>list
-----
+Within the `<INSTALL_DIRECTORY>`, a directory is created named ${branding-lowercase}-${project.version}.
+This directory will be referred in the documentation as `<${branding}_HOME>`.
 
-// Verify this step is still required.
-* If the ${branding} Standalone Solr Server will be installed later, an additional configuration step is required for the ${branding} kernel.
-Add the following lines to the bottom of the `<INSTALL_DIR>/etc/org.ops4j.pax.web.cfg` file:
+. Check the available storage space on the system to ensure the deployment will not exceed the available space.
+. Set maximum storage space on the `<${branding}_HOME>/deploy` and `<${branding}_HOME>/system` directories to restrict the amount of space used by deployments.
+. Do not assume the deployment is from a trusted source; verify its origination.
 
-.Additional Configuration Step
-----
-# Jetty Configuration
-`org.ops4j.pax.web.config.file=<KARAF.HOME>/etc/jetty.xml`
-----
+==== Directory Permissions
+
+Restrict access to sensitive files by ensuring that the only users with access privileges are administrators.
 
-* Run the ${branding} using the appropriate script.
+===== Directory Permissions on Windows
 
-.*NIX
-----
-<INSTALL_DIRECTORY>/${branding-lowercase}-${project.version}/bin/${branding-lowercase}
-----
+Set directory permissions to protect the ${branding} from unauthorized access.
 
-.Windows
-----
-<INSTALL_DIRECTORY>/${branding-lowercase}-${project.version}/bin/${branding-lowercase}.bat
-----
+. Right-click on the file or directory noted below then select *Full Control -> Administrators -> System*.
+. Click *Properties -> Security -> Advanced* and select `Creator Owner` for `<${branding}_HOME>` (e.g., `C:\${branding-lowercase}`).
+. Restrict access to sensitive files by ensuring that only *System* and *Administrators* have Full Control to the below files by right-clicking on the file or directory below then selecting *Properties -> Security -> Advanced*
+. Delete any other groups or users listed with access to `<${branding}_HOME>/etc` and `<${branding}_HOME>/deploy`.
 
-The distribution takes a few moments to load depending on the hardware configuration.
-Execute the following command at the command line for status:
+===== Directory Permissions on *NIX
 
-.View Status
-----
-${branding}${at-symbol}local>list
-----
-The list of bundles should look similar to this:
+Set directory permissions to protect the ${branding} from unauthorized access.
 
-.${branding} List of Apps Installed
-----
-.${branding}${at-symbol}local>list
-START LEVEL 100 , List Threshold: 50
-   ID   State         Blueprint      Spring    Level  Name
-[ 111] [Active     ] [            ] [       ] [   80] Commons IO (2.1.0)
-[ 113] [Active     ] [Created     ] [       ] [   80] ${branding} :: Distribution :: Console Branding Plugin (${project.version})
-----
+* Change ownership of ${branding}_HOME
+** `chown -R NON_ROOT_USER <${branding}_HOME>`
```

Comment:
In `installing-intro-contents.adoc`, we use `${branding-lowercase}-user`. Should we be consistent and use the same thing in both places?